testsuite/cluster-test/src/cluster_swarm/cluster_swarm_kube.rs

```diff
@@ -264,6 +264,12 @@ impl ClusterSwarmKube {
         Ok(())
     }
 
+    async fn list_nodes(&self) -> Result<Vec<Node>> {
+        let node_api: Api<Node> = Api::all(self.client.clone());
+        let lp = ListParams::default().labels("nodeType=validators");
```

Comment:
Can we take the `labels` as a parameter. I imagine we will run different nodeTypes when we start running LSR/Vault

Reply:
Will need to think more about splitting node types later.
Some of the usage need 'all nodes cluster test created nodes' and some need 'nodes of type x'